app/src/androidTest/java/org/mozilla/fenix/ui/DownloadTest.kt

```diff
@@ -56,35 +52,20 @@ class DownloadTest {
         }
     }
 
-    @Suppress("Deprecation")
     @After
     fun tearDown() {
         mockWebServer.shutdown()
 
-        // Clear Download
-        runBlocking {
-            val downloadedFile = File(
-                Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS),
-                "Globe.svg.html"
```

Comment:
I remember adding .svg.html because there was a problem saving the svg. It might have been related to the MIME type issues we had with Mockwebserver a while back. I have no idea if it's still needed or not. Does the above approach work with just passing in the filename?

Reply:
It worked so far in my trial runs.  Did it fail to delete it consistently in your case?

Reply:
It was on my Pixel 2 physical device. Maybe it's not an issue anymore. I trust you that it works now.

Reply:
Tested and it works well on a physical Pixel 2 as well. Merging.